debezium-connector-mysql/pom.xml

```diff
@@ -74,6 +74,22 @@
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-avro-converter</artifactId>
         </dependency>
+        <!-- Used for unit testing with Docker -->
```

Comment:
Ok, it's my hope that we'd eventually be using only one plug-in, but we can do it iteratively. The less moving parts, the better :)

> Still the configuration can be cleaned to keep onyl build section and from runtime section only container naming.

+1 That sounds good.